rest-countries-API/script.js

```diff
@@ -1,28 +1,21 @@
 const countriesContainer = document.querySelector('.countries-container')
+
 fetch("https://restcountries.com/v3.1/all")
   .then((resolve) => resolve.json())
   .then((data) => {
     data.forEach((country) => {
       const countryCard = document.createElement("a");
       countryCard.classList.add("country-card");
-      const cardHTML = `
-      <img src="https://flagcdn.com/is.svg" alt="flag" />
-      <div class="card-text">
-      <h3 class="card-title">Iceland</h3>
-      <p><b>Population: </b>81,770,900</p>
-      <p><b>Region: </b>Europe</p>
-      <p><b>Capital: </b>Berlin</p>
-      </div>
-      `
+      const cardHTML =
+          `<img src="${country.flags.svg} " alt="flag" />
+          <div class="card-text">
+          <h3 class="card-title">${country.name.common}</h3>
+          <p><b>Population: </b>${country.population.toLocaleString('en-IN')}</p>
```

Comment:
used `toLocaleString('en-IN')` to format the population numbers in Indian format for readability.